### PR TITLE
Update: rock-paper-scissors example

### LINKED ISSUE #77: Update other games to work with the refactored code
## Description
Currently only the examples/simple works. We at least need the hockey game to work again.

### PR DESCRIPTION
**Pull request by SveaGaenzle**
I updated the `rockpaperscissors` example to work with the refactored code.

## Issue:

- resolves #77 

## Description:
- I added a config file.
- I updated the `agent` so that it always returns a random int between 0 and 2 as a step and prints the results at the end of the game. As in the other agents, you can also select a token from the four dummy users.
- I renamed the file `server` to `game` and refactored the code to work with the new version.

### CODE REVIEW

File: examples/rockpaperscissors/agent.py [hunk 1]
```diff
@@ -1,17 +1,27 @@
-from comprl.client.agent import COMPAgent
+from comprl.client import Agent
+import random
 
 
-class RPSAgent(COMPAgent):
-    """Dummy Agent for testing. Returns random action"""
+RPSAgent = Agent()
```
Comment (Raining-Cloud) [inline examples/rockpaperscissors/agent.py:5]: RPSAgent is an object at this point it should be something like agent or so (see [here](https://google.github.io/styleguide/pyguide.html#s3.16-naming ))

File: examples/rockpaperscissors/game.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+from enum import IntEnum
+
+from comprl.server.interfaces import IGame, IPlayer
+from comprl.shared.types import PlayerID
+
+Sign = IntEnum("Sign", ["ROCK", "PAPER", "SCISSORS"])
+
+
+class RPSServer(IGame):
```
Comment (Raining-Cloud) [inline examples/rockpaperscissors/game.py:16]: This is the RPSGame not a server

**Review by Raining-Cloud — CHANGES_REQUESTED** [review]

File: examples/rockpaperscissors/game.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+class RPSGame(IGame):
+    """
+    This class represents a rock-paper-scissors game.
+    """
+
+    def __init__(self, players: list[IPlayer]) -> None:
+        super().__init__(players=players)
+
+        self.scores: dict[PlayerID, float] = {p.id: 0.0 for p in players}
```
Comment (Cari1111) [inline examples/rockpaperscissors/game.py:24]: This is unnecessary because `self.scores` is already initialized in this way.

**Review by Cari1111 — CHANGES_REQUESTED** [review]
Thank you for updating this example game! The game works correctly and the code is written well. I just have two little suggestions below.

**Review by Cari1111 — APPROVED** [review]
The changes I requested are correctly implemented and the game works!

**Review by Raining-Cloud — APPROVED** [review]
LGTM!

**Pull request merged**